test/unit/hostgroup_test.rb

```diff
@@ -357,14 +357,41 @@ def setup_user(operation)
     assert_equal "#{hostgroup.id}-a-b",  hostgroup.to_param
   end
 
-  test "clone should clone config groups as well" do
-    group = FactoryGirl.create(:hostgroup, :name => 'a')
-    config_group = ConfigGroup.create!(:name => 'Blah')
-    group.config_groups << config_group
-    group.save
-
-    cloned = group.clone("new_name")
-    assert cloned.config_groups.include?(config_group)
+  context "#clone" do
+    let(:group) {FactoryGirl.create(:hostgroup, :name => 'a')}
+
+    test "clone should clone config groups as well" do
+      config_group = ConfigGroup.create!(:name => 'Blah')
+      group.config_groups << config_group
+      group.save
```

Comment:
yes it is. removed it to check, #clone goes to the db in some of the fields, and in the cloned group we REALLY need the save, because we need to make sure everything actually gets saved to the db and is retrievable.